### PR TITLE
Addition of aria-colcount expectations, test file, results

### PR DESCRIPTION
**Pull request by RavenAlly**

### CODE REVIEW

**Comment by RavenAlly** [comment]
I've since realised I've made a couple of typos, I'll push a few updated lines tomorrow.

**Comment by RavenAlly** [comment]
Updates done, ready for review / merge

File: data/tech/aria/aria-colcount_attribute.json [hunk 1]
```diff
@@ -9,6 +9,46 @@
       "url": "https://www.w3.org/TR/wai-aria-1.1/#aria-colcount"
     }
   ],
-  "assertions": [],
-  "date_updated": "2019-01-06"
+  "assertions": [
+    {
+      "id": "convey_column_count_all_cols",
```
Comment (mfairchild365) [inline data/tech/aria/aria-colcount_attribute.json:14]: I'm wondering if we can remove this expectation. See my similar comment for `aria-rowcount` for context.

File: data/tech/aria/aria-colcount_attribute.json [hunk 2]
```diff
@@ -9,6 +9,46 @@
...
+      },
+      "css_target": "table[aria-colcount]",
+      "operation_modes": [
+        "sr/reading"
+      ]
+    },
+    {
+      "id": "do_not_calculate_unknown_column_count",
+      "title": "convey a column count when total column count is unknown",
+      "rationale": "Screen readers should not attempt to calculate the column count when it is unknown",
```
Comment (mfairchild365) [inline data/tech/aria/aria-colcount_attribute.json:42]: Also wondering if we should call out the "-1" value here at all (similar to my comment for aria-rowcount)

File: data/tests/html/aria/aria-colcount.html [hunk 3]
```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html>
```
Comment (mfairchild365) [inline data/tests/html/aria/aria-colcount.html:2]: Would be good to have a lang attribute

**Review by mfairchild365 — COMMENTED** [review]
Awesome work on this too! Just a few comments/questions, similar to the ones I left for the aria-rowcount PR.

File: data/tests/html/aria/aria-colcount.html (hunk 3 above)
Comment (RavenAlly) [reply data/tests/html/aria/aria-colcount.html]: Added. ✔️

File: data/tech/aria/aria-colcount_attribute.json (hunk 2 above)
Comment (RavenAlly) [reply data/tech/aria/aria-colcount_attribute.json:29]: Added verbatim note from ARIA spec. ✔️

File: data/tech/aria/aria-colcount_attribute.json (hunk 1 above)
Comment (RavenAlly) [reply data/tech/aria/aria-colcount_attribute.json]: Removed the all_cols expectation and test results. ✔️

**Comment by mfairchild365** [comment]
Thank you again!

**Pull request merged**